src/os_user_view.eliomi

```diff
@@ -286,12 +286,6 @@ val disconnect_all_link
     @param text_link link text.
  *)
 
-val disconnect_button
-  :  ?text_logout:string
-  -> unit
-  -> [> Html_types.li_content_fun] Eliom_content.Html.F.elt
-(** A disconnect button *)
-
```

Comment:
Unwanted change?

Reply:
No, the signature appeared twice in the same file.